src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -150,15 +150,6 @@ public partial interface IPublicClientApplication : IClientApplicationBase
         /// </summary>
         /// <remarks>This API may change in future release.</remarks>
         string iOSKeychainSecurityGroup { get; set; }
-
-        /// <summary>
-        /// Xamarin iOS specific property enabling the application to share the token cache with other applications sharing the same keychain security group.
-        /// If you use this property, you MUST add the capability to your Application Entitlement.
-        /// When using this property, the value must contain the TeamId prefix, which is why this will be obsolete in future releases.
-        /// </summary>
-        /// <remarks>This API will be removed in MSAL v3.x. See https://aka.ms/msal-net-ios-keychain-security-group for details</remarks>
-        [Obsolete("Use iOSKeychainSecurityGroup instead (See https://aka.ms/msal-net-ios-keychain-security-group)", false)]
-        string KeychainSecurityGroup { get; set; }
```

Comment:
was this added as migration aid instead?

Reply:
#no....but will include

Reply:
have it here: [https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/817](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/817)

---
In reply to: [249953245](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/814#discussion_r249953245) [](ancestors = 249953245)